<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandBox</class>
 <widget class="QWidget" name="CommandBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hermes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color:#0D0221;
}


   QLineEdit
{
    border:7px ridge #B3E5FC;
                                            border-radius:15px;
                                            font-size:15px;
                                            color:#05D9E8;
                                            background-color:black;
}
QPushButton
{
    border:7px ridge #FF2A6D;
                                               border-radius:15px;
                                               font-size:15px;
                                               color:#05D9E8;
                                               background-color:black;
}

QPushButton:hover
{
    border:7px ridge #FF2A6D;
                                               border-radius:15px;
                                               font-size:30px;
                                               color:#00FF00;
                                               background-color:black;
}

QTextEdit
{
    border:7px ridge #8A2BE2;
                                            border-radius:15px;
                                            font-size:15px;
                                            color:#00FF00;
                                            background-color:black;
}

QLabel
{
     font-size:20px ;
     font-style: italic;
                                             color:white;
                                             background-color:black;


}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
     <item>
      <widget class="QTextEdit" name="outputArea">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="status">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1">
         <item>
          <widget class="QLineEdit" name="inputField">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="placeholderText">
            <string>请输入指令...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="executeButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>执行</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>就绪</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
